SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp

```diff
@@ -224,6 +230,12 @@ void ULayeredLBStrategy::SetVirtualWorkerIds(const VirtualWorkerId& FirstVirtual
 	}
 }
 
+bool ULayeredLBStrategy::CouldHaveAuthority(const TSubclassOf<AActor> Class) const
+{
+	check(IsReady());
+	return *VirtualWorkerIdToLayerName.Find(LocalVirtualWorkerId) == GetLayerNameForClass(Class);
+}
```

Comment:
I agree that having this makes sense but I think we need to rethink our presentation from a high level since now we have should/can/could semantics together

Reply:
This is purely temporary and I want to delete it.  (I've got a comment about that in the header, but I'll add one here as well.)   I have to have it because there is 1 place in the Scavengers code base that uses the 1 place in SpatialStatics that this is called. Once we get them converted over, we can remove this.

Reply:
ah 👍